ext/nnstreamer/tensor_filter/tensor_filter_openvino.cc

```diff
@@ -632,7 +661,7 @@ ov_open (const GstTensorFilterProperties * prop, void **private_data)
   }
 
   tfOv = static_cast<TensorFilterOpenvino *>(*private_data);
-  if (tfOv != nullptr) {
+  if (tfOv != nullptr && tfOv->isModelLoaded ()) {
```

Comment:
tfOv != null and not-loaded, this will not delete the instance.

Reply:
IMO, it is better to move 'tfOv->isModelLoaded ()' before checking the model path (line 665)

Reply:
@jaeyun-jung Modified. Thanks!